tests/testsrc/test262.properties

```diff
@@ -2189,7 +2188,8 @@ built-ins/String 140/1182 (11.84%)
     prototype/indexOf/searchstring-tostring-errors.js
     prototype/indexOf/searchstring-tostring-toprimitive.js
     prototype/indexOf/searchstring-tostring-wrapped-values.js
-    prototype/isWellFormed 8/8 (100.0%)
+    prototype/isWellFormed/not-a-constructor.js {unsupported: [Reflect.construct]}
+    prototype/isWellFormed/to-string-primitive.js
```

Comment:
Due to missing Symbol.toPrimitive I presume

Reply:
It actually fails with a syntax error beforehand because it uses for const with destructuring. Meant to mention that before but forgot.

Reply:
Ah, another one of our challenges 🫣